modules-themes/readme.md

```diff
@@ -0,0 +1,42 @@
+# Listing third party open source and free of charge modules and themes for the PrestaShop e-commerce solution
+
+There are plenty of useful themes and modules available on the web from third party developers and company for the PrestaShop e-commerce solution. But unfortunately, there is no place currently to list them and find them.
+
+So, let's build a knowledge base. If you want to contribute to this page, please create a pull request with your modules and themes, following the example:
```

Comment:
Are you sure about this? People will just promote their stuff even if it's shitty

Reply:
it's a pull request, so it's possible to say "no thanks". it will be possible to check in the module repository if they added a licence file, how they handle dependencies (if any), etc.
if it's really shitty, it will be possible to tell it in the pull request.
if it is possible to set a few criteria, then, it is possible to add them in the introduction of this document.

Reply:
There are some moderation works as awesome-for-beginners does yes